### ISSUE TITLE
Update latest print proofs (A1 size)

### ISSUE THREAD

**Issue by davelab6** [TypeNetwork/Roboto-Flex#135]
I'm ready to go to a printshop tomorrow morning but I see https://github.com/TypeNetwork/Roboto-Flex/tree/master/docs/proof-RobotoFlex hasn't been updated with the proofs we discussed. No pressure but when do you think you'll be able to update them, with a README with print production tips? :)

**Comment by sannorozco** [comment]
Yes Dave, I’m updating these rn, will b updated soon and ready for you.

**Comment by sannorozco** [comment]
Proofs updated!

opsz8: Two sizes, A1 showing all glyphs and spacing, Tabloid showing text at optical size, & kerning

opsz14: Two sizes, A1 showing all glyphs and spacing, Tabloid showing text at optical size, & kerning

opsz144: One size, A1 showing all glyphs and spacing, text at optical size, & kerning, each one by width. **(There’s almost 1.5k pages in A1 size for this optical size, PLEASE consider before printing you'll need help moving these, if necessary can be scaled down)**

For printing at 100%, at least 1000 ppi.

cc @dberlow

**Comment by davelab6** [comment]
Alas. I went to the print shop and they said actually they can only do

> 12x18" on regular paper (Xerox machine) and in the wide format plotter in photo paper 48"

(This is USA so everything is in inches)

A1 is 23.4 x 33.1 inch, roughly half, and the pages are 2-up spread like, so maybe it's best to repaginate like that

**Comment by sannorozco** [comment]
Proofs updated.

Proofs packages for RobotoFlex 

[opsz8](https://github.com/TypeNetwork/Roboto-Flex/tree/master/docs/proof-RobotoFlex/opsz8): One size 18x12 inches, showing all glyphs and spacing, text at optical size, & kerning

[opsz14](https://github.com/TypeNetwork/Roboto-Flex/tree/master/docs/proof-RobotoFlex/opsz14): One size 18x12 inches, showing all glyphs and spacing, text at optical size, & kerning

[opsz144](https://github.com/TypeNetwork/Roboto-Flex/tree/master/docs/proof-RobotoFlex/opsz144) One size 48x23 inches, showing all glyphs and spacing, text at optical size, & kerning, each one by width. Recommend printing only [RobotoFlex-opsz144-proofs-glyphs-48.pdf](https://github.com/TypeNetwork/Roboto-Flex/tree/master/docs/proof-RobotoFlex/opsz144/RobotoFlex-opsz144-proofs-glyphs-48.pdf) (There’s a bit more 1k pages in 48x23 inches size for this optical size, PLEASE consider before printing you'll need help moving these) Or you can review spacing and kern on screen

**Comment by davelab6** [comment]
> One size 48x23 inches

I'm unable to get that printed at all. 

I think what I need is a very simple set of proofs at 144pt for all masters ASCII, glyph table style, and then 3 more pages - all caps, title case and sentence case - in English, to start.

**Comment by sannorozco** [comment]
Added proof size 18x12 inches, showing ascii, title case, sentence case, & all caps. Type at 144 is too big, so I added more than 3 pages, you can delete some of them if you think necessary.

(thank you for your patience, I’m on vacation)
